SCHEMA/transaction.schema.js

```diff
@@ -11,9 +13,17 @@ const transactionSchema = new Schema({
         type: Number, 
         required: [true, "Cannot make a transaction without an ammount"]
     },
-    name_on_transaction:{
+    account_number: {
+        type: String,
+        required: [true, "Cannot make a transaction without account number"]
+    }, 
+    fname:{
+        type: String,
+        required:[true, "A first name must be provided"]
+    },
+    lname:{
         type: String,
-        required:[true, "A name must be provided"]
+        required:[true, "A last name must be provided"]
     },
     transaction_status:{
```

Comment:
how will we use this. when will "pending" status change

Reply:
Not sure at the moment..just having it here for future use